From 11dfdbd20a84f9399459098ae5d435b119995072 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Tue, 6 Aug 2024 15:45:13 +0000
Subject: [PATCH] util: Specify Gtk version

Some utils are imported by ocrfeeder-cli, and the version is not specified before in this case.
---
 src/ocrfeeder/util/lib.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/ocrfeeder/util/lib.py b/src/ocrfeeder/util/lib.py
index cd4ab95..84f5eb1 100644
--- a/src/ocrfeeder/util/lib.py
+++ b/src/ocrfeeder/util/lib.py
@@ -17,6 +17,8 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ###########################################################################
 
+import gi
+gi.require_version('Gtk', '3.0')
 import math
 import mimetypes
 import os
-- 
GitLab

